verify_rpms/rpm_verifier.py

```diff
@@ -44,85 +47,83 @@ def get_rpmdb(container_image: str, target_dir: Path, runner: Callable = run) ->
     return target_dir
 
 
-def get_unsigned_rpms(rpmdb: Path, runner: Callable = run) -> list[str]:
-    """
-    Get all unsigned RPMs from RPM DB path
-    Filter and return the unsigned RPMs
-    :param rpmdb: path to RPM DB folder
-    :param runner: subprocess.run to run CLI commands
-    :return: list of unsigned RPMs within the folder
-    """
+def get_rpms_data(rpmdb: Path, runner: Callable = run) -> list[str]:
+    """Get RPMs data from RPM DB"""
     rpm_strs = runner(
         [
             "rpm",
             "-qa",
             "--qf",
-            "%{NAME}-%{VERSION}-%{RELEASE} %{SIGGPG:pgpsig} %{SIGPGP:pgpsig}\n",
```

Comment:
What is this change about?

Reply:
To get the key ID (or no key) we only need `SIGPGP`